### PR TITLE
Add Chinese version README-CN.md

### PR DESCRIPTION
**Pull request by ziyue5**
# Description
Add Chinese version README in README-CN.md and add a hyperlink in English version README.md.

---

Reviewing this pull request? Check out our [Code Review Practices](https://github.com/ifmeorg/ifme/wiki/Code-Review-Practices) guide if you haven't already!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
...
 
-README in: [Spanish](https://github.com/ifmeorg/ifme/blob/main/README-ES.md), [Portuguese](https://github.com/ifmeorg/ifme/blob/main/README-PT.md), [French](https://github.com/ifmeorg/ifme/blob/main/README-FR.md), [Korean](https://github.com/ifmeorg/ifme/blob/main/README-KO.md), [Indonesian](https://github.com/ifmeorg/ifme/blob/main/README-ID.md), [Turkish](https://github.com/ifmeorg/ifme/blob/main/README-TR.md), [Sinhala](https://github.com/ifmeorg/ifme/blob/main/README-LK.md)
+README in: [Spanish](https://github.com/ifmeorg/ifme/blob/main/README-ES.md), [Portuguese](https://github.com/ifmeorg/ifme/blob/main/README-PT.md), [French](https://github.com/ifmeorg/ifme/blob/main/README-FR.md), [Korean](https://github.com/ifmeorg/ifme/blob/main/README-KO.md), [Indonesian](https://github.com/ifmeorg/ifme/blob/main/README-ID.md), [Turkish](https://github.com/ifmeorg/ifme/blob/main/README-TR.md), [Sinhala](https://github.com/ifmeorg/ifme/blob/main/README-LK.md), [Chinese](https://github.com/ifmeorg/ifme/blob/main/README-CN.md)
```
Comment (julianguyen) [inline README.md:10]: We'll want to add a link to Chinese on the other language READMEs too!

**Review by julianguyen — CHANGES_REQUESTED** [review]
Thanks so much for taking the time to do this!

File: README.md (hunk 1 above)
Comment (ziyue5) [reply README.md:10]: Sure! I have added them.

**Review by julianguyen — APPROVED** [review]
Thank you again! :D

**Pull request merged**